templates/vncserver@.service

```diff
@@ -52,6 +55,13 @@ ExecStartPre=/bin/sh -c "/usr/bin/vncserver -kill :%i > /dev/null 2>&1 || :"
 #   https://github.com/cisagov/ansible-role-vnc-server/issues/5
 ExecStart=/usr/bin/vncserver :%i -localhost no -alwaysshared -fg -geometry 1920x1080 -SecurityTypes VncAuth
 ExecStop=/usr/bin/vncserver -kill :%i
+# Sometimes BurpSuitePro runs out of memory and is killed by the
+# kernel, bringing down this service in the process.  This doesn't fix
+# that issue, but it does automatically restart the service so that
+# the assessors can continue on without asking dev folks to intervene.
```

Comment:
Just a thought, but is there a more catch-all term to use instead of "dev folks" as we will hopefully move away from care and feeding of environments?

Reply:
How about this?
```suggestion
# the assessors can continue on without needing anyone else to intervene.
```

Reply:
Perfect!